### PR TITLE
[backport] Update variable names for consistent naming convention

### PR DESCRIPTION
**Pull request by takagi**
Backport of #8074

### CODE REVIEW

**Comment by takagi** [comment]
[automatic post] Jenkins, test this please.

**Comment by takagi** [comment]
I removed the following files from backporting as `v6` branch does not have:
- deleted by us:   examples/chainermn/cifar/models/VGG.py
- deleted by us:   examples/chainermn/cifar/train_cifar.py
- deleted by us:   tests/chainerx_tests/unit_tests/routines_tests/test_reduction.py

File: chainer/distributions/independent.py [hunk 1]
```diff
@@ -236,11 +236,11 @@ def _kl_independent_independent(dist1, dist2):
     # KL(p || q) and do a `reduce_sum` on the reinterpreted batch dimensions.
     if dist1.event_shape == dist2.event_shape:
         if p.event_shape == q.event_shape:
-            num_reduce_dims = len(dist1.event_shape) - len(p.event_shape)
-            reduce_dims = tuple([-i - 1 for i in range(0, num_reduce_dims)])
+            n_axes = len(dist1.event_shape) - len(p.event_shape)
+            n_reduce_dims = tuple([-i - 1 for i in range(0, n_axes)])
```
Comment (emcastillo) [inline chainer/distributions/independent.py]: Maybe I am mistaken but this is not a num but a tuple, right?

Comment (takagi) [reply chainer/distributions/independent.py]: Yes, but this PR is a backport of #8074, let us leave it as is here.

Comment (emcastillo) [reply chainer/distributions/independent.py]: Ok Lets do that and fix both in another PR :)

**Pull request merged**